.github/workflows/release.yaml

```diff
@@ -17,9 +17,19 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.9
+    - name: Install apt dependencies
+      run: |
+        sudo apt-add-repository ppa:ubuntugis/ubuntugis-unstable
+        sudo apt-get update
+        sudo apt-get install gdal-bin libgdal-dev
     - name: Install pip dependencies
-      run: pip install .[tests]
+      run: |
+        pip install gdal tqdm  # TODO: these deps shouldn't be needed
+        pip install .[datasets,tests,train]
+        pip install -r docs/requirements.txt
     - name: Run notebook checks
+      env:
+        MLHUB_API_KEY: ${{ secrets.MLHUB_API_KEY }}
```

Comment:
Alternatively, we could use a different dataset that doesn't require an API key to download

Reply:
I'd rather not have an API key secret, and would rather use UCMerced instead of Cyclone. Lets put that on the 0.2 release path.